pkg/operator/ceph/file/controller.go

```diff
@@ -200,7 +200,8 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 		// This handles the case where the operator is not ready to accept Ceph command but the cluster exists
 		if !cephFilesystem.GetDeletionTimestamp().IsZero() && !cephClusterExists {
 			// Remove finalizer
-			err := opcontroller.RemoveFinalizer(r.client, cephFilesystem)
+			finalizerName := opcontroller.BuildFinalizerName(cephFilesystem.GetObjectKind().GroupVersionKind().Kind)
```

Comment:
same finalizerName initialized twice in line 203 and 262.  We can do it once and use it.  Same for other daemon controllers as well.

Reply:
Changed a pattern, by using another function 
Thanks for this sugestion.